hummingbot/connector/exchange/binance/binance_auth.py

```diff
@@ -48,6 +48,7 @@ def add_auth_to_params(self,
 
         request_params = OrderedDict(params or {})
         request_params["timestamp"] = timestamp
+        request_params['recvWindow'] = 5000
```

Comment:
This one is not necessary. In the docs says that if you don't specify the recvWindow will be 5000 by default.
Note: Use double quotes (btw, I'm always using single quotes haha but I run a find and replace after submit to follow the standards)
<img width="683" alt="image" src="https://user-images.githubusercontent.com/36869960/188498216-2fe9f3e9-da7e-42ff-82cf-dde0f5b898a0.png">

Reply:
Understood, I had thought perhaps this could benefit someone who wanted to change this value from the default